evi-next-js-function-calling/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "evi-next-js-function-calling",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@humeai/voice": "0.1.3-beta.1",
+    "@humeai/voice-react": "0.1.3-beta.1",
```

Comment:
@hume-brian Can you confirm whether these package versions are publicly available? (private vs public NPM registry) 

We'll want to make sure the packages are installable by external devs attempting to run the project locally.

Reply:
all packages are now on 0.1.3 as of this morning